it/src/main/java/org/corfudb/universe/group/Group.java

```diff
@@ -0,0 +1,68 @@
+package org.corfudb.universe.group;
+
```

Comment:
For now it seems like we just need cluster. Can you add this when its needed?

Reply:
No, having groups is very important, the next steps will be to use groups for corfuClients, 
This is a requirement we have right now